apps/site/assets/css/_page.scss

```diff
@@ -11,6 +11,27 @@
       border-bottom-width: 0;
     }
   }
+
+  .list-group-item {
+    border-top: unset;
+    border-top: solid $gray-lightest;
+    border-width: 1px 0 0 0;
+  }
+
+  .m-event__title {
+    a {
+      color: $black;
+      &:hover {
+        color: $brand-primary;
+      }
+    }
+  }
+
+  .m-event__date-circle {
+    @include media-breakpoint-down(sm) {
+      display: none;
+    }
+  }
 }
```

Comment:
I think we might need to add an extra rule to adjust the padding on `.m-event` on smaller breakpoints, currently it shows a bit too narrow:

<img width="329" alt="image" src="https://user-images.githubusercontent.com/2136286/174673556-76f8e9e6-ba41-41dc-81ad-198827a416b4.png">